packages/server/graphql/types/User.ts

```diff
@@ -525,17 +526,22 @@ const User: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<any, GQLC
           description:
             'The meetingId to check for the invitation, if teamId not available (e.g. on a meeting route)'
         },
+        meetingSeriesId: {
+          type: GraphQLID,
+          description:
+            'The meetingSeriesId to check for the invitation, if teamId not available (e.g. on a meeting series route)'
+        },
```

Comment:
+1 where do we use it? do you mind sharing a little bit more what's the use case here? having troubles understanding what's the problem we're solving 🙏